go/pkg/bertyprotocol/store_metadata.go

```diff
@@ -286,6 +286,31 @@ func metadataStoreAddEvent(ctx context.Context, m *metadataStore, g *bertytypes.
 	return op, nil
 }
 
+// ContactReady informs if secrets are sent to the member, so if it can read messages send by us
```

Comment:
```suggestion
// ContactReady waits until the secrets are sent to the member.
```


not sure whether it's "Contact" or "Member"